tools/eslint-plugin-azure-sdk/src/rules/github-source-headers.ts

```diff
@@ -1,30 +1,28 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Just a thought, usually when doing PRs that are entirely mechanical (e.g. insert copyright headers everywhere or run the formatter on all files), it's better to do those changes as their own separate PR to avoid having to skim through every file to see if substantive changes have happened. 

I'm not worried about it for this review, but would appreciate it in future PRs. 😄